content/md/en/docs/reference/command-line-tools/node-template.md

```diff
@@ -147,86 +147,125 @@ Use the `node-template benchmark` command to analyze the resources required to e
 | `-p`, `--pallet <pallet>` | Specifies the FRAME pallet to benchmark, or `*` to benchmark all pallets. If you benchmark all pallets, you must also specify `--extrinsic *` to benchmark all extrinsic calls.
-| `-r`, `--repeat <repeat>` | Specifies the number of times to repeat the execution of a benchmark from within the Wasm binary. The default is one execution.
+| `--record-proof` | Estimates the proof-of-validation (PoV) size.
+| `-r`, `--repeat <repeat>` | Specifies the number of times to repeat the execution of a benchmark from within the WebAssembly binary. The default is one execution.
 | `-s`, `--steps <steps>` | Specifies how many samples to take across the variable components. The default is one sample.
 | `--template <template>` | Specifies the path to a Handlebars template file used for outputting benchmark results.
-| `--tracing-receiver <receiver>` | Specifies the receiver to process tracing messages. The default is Log.
-| `--tracing-targets <targets>` | Sets a custom profiling filter. Syntax is the same as for logging: `<target>=<level>`.
-| `--wasm-execution <method>` | Specifies the method for executing Wasm runtime code. Valid values are `interpreted`, or `compiled`. The default is `Compiled`.
 
-#### Examples
+For examples of different benchmarking subcommands and the related command-line options, see [Benchmarking examples](#benchmarking-examples).
+
+#### Benchmarking examples
 
-To start a node and benchmark all of the extrinsic calls in all of the pallets, you might a command similar to the following:
+After you have compiled the runtime with benchmarking enabled, you can run a command similar to the following to benchmark all of the function calls in all of the pallets that have runtime-benchmarking configured:
 
 ```shell
-./target/release/node-template benchmark \
+./target/release/node-template benchmark pallet \
     --chain dev \
-    --execution wasm \
-    --wasm-execution compiled \
-    --extrinsic '\*' \
-    --pallet '\*' \
-    --output ./pallets/example/weights.rs
+    --extrinsic "*" \
+    --pallet "*" \
+    --output pallets/weights.rs
+```
+
+With this command, each function call is executed once with a single value and the resulting weight is recorded in the `weights.rs` file.
+
+Depending on the function you want to benchmark, you can add the `--steps` and `--repeat` command-line options to execute the call multiple times with different values.
+For example, the following command executes the `do_something` function in the `pallet_template` and calls the function 20 times to take 10 data points:
+
+```shell
+./target/release/node-template benchmark pallet \
+    --chain dev \
+    --extrinsic do_something \
+    --pallet pallet_template \
+    --steps 10 \
+    --repeat 20 \
+    --output pallets/weight2.rs
+```
```

Comment:
You probably already did it in another spot; but I think can add some output of how the stuff should look like.  
The `--list` from above produces:  
```pre
pallet, benchmark
frame_benchmarking, addition
frame_benchmarking, subtraction
...
pallet_timestamp, set
pallet_timestamp, on_finalize
pallet_template, do_something
```

and this one:
<details>
  <summary>spoiler</summary>

```pre
Pallet: "pallet_template", Extrinsic: "do_something", Lowest values: [], Highest values: [], Steps: 10, Repeat: 20
Raw Storage Info
========
Storage: TemplateModule Something (r:0 w:1)

Median Slopes Analysis
========
-- Extrinsic Time --

Model:
Time ~=     8.08
    + s        0
              µs

Reads = 0 + (0 * s)
Writes = 1 + (0 * s)

Min Squares Analysis
========
-- Extrinsic Time --

Data points distribution:
    s   mean µs  sigma µs       %
    0     11.73     0.942    8.0%
   11     7.914     0.077    0.9%
   22     7.918     0.095    1.1%
   33     7.989      0.08    1.0%
   44     8.034     0.095    1.1%
   55     7.918      0.07    0.8%
   66     8.198      0.16    1.9%
   77     8.052     0.088    1.0%
   88     8.078     0.065    0.8%
  100     7.976     0.135    1.6%

Quality and confidence:
param     error
s         0.003

Model:
Time ~=    9.235
    + s        0
              µs

Reads = 0 + (0 * s)
Writes = 1 + (0 * s)
```

</details>

Reply:
Added --list output and a sample output